BigCommerce/UserInterface/01a_Connections.md

```diff
@@ -0,0 +1,198 @@
+
+- [DISABLE]  
+    Click this button to disable the selected connection(s). This button is only displayed if the checkbox of at least one active connection is selected.
+
+- [ENABLE]  
+    Click this button to enable the selected connection(s). This button is only displayed if the checkbox of at least one inactive connection is selected. 
+
+- ![Edit](../../Assets/Icons/Edit01.png "[Edit]") (Edit)  
+    Click this button to edit the selected *BigCommerce* connection. This button is only displayed if a single checkbox in the list of connections is selected. Alternatively, you can click directly a row in the list to edit a connection. The *Edit connection* view is displayed, see [Edit BigCommerce connection](#edit-bigcommerce-connection).
+
+The list displays all available connections. Depending on the settings, the displayed columns may vary. All fields are read-only.
+
+- *Name*  
+    Connection name.
+
+- *Status*  
+    Connection status. The following statuses are available:
+    - ![Status](../../Assets/Icons/Status01.png "[Status]") **Active**   
+        The connection is enabled and data is being synchronized via the connection.
+    - ![Status](../../Assets/Icons/Status04.png "[Status]") **Inactive**   
+        The connection is disabled and no data is being synchronized via the connection.   
+
+- *Order/Return have errors*  
+    Indication whether errors have occurred in orders or returns. The following options are available:  
+    - **Yes**   
+        Errors have occurred.   
+    - **No**   
+        No errors have occurred.
+
+- *Driver*  
+    Driver name.
+
+- *ID*  
+    Connection identification number. The ID number is automatically assigned by the system.
+
+- ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add)  
+    Click this button to add a *BigCommerce* connection. The *Create connection* view is displayed, see below.
+
+
+
+## Create BigCommerce connection
+
+*Omni-Channel > Settings > Tab CONNECTIONS > Button Add*
+
+![Create connection](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/CreateConnection.png "[Create connection]")
+
+- ![Back](../../Assets/Icons/Back02.png "[Back]") (Back)   
+    Click this button to close the *Create connection* view and return to the connection list. All changes are rejected.
+
+- *Name*   
+    Enter a connection name.
+
+-  ![Edit](../../Assets/Icons/Edit02.png "[Edit]") (Edit)  
+    Click this button to edit the connection name. This button is only displayed if the connection name has been confirmed. 
+
+- ![Apply](../../Assets/Icons/Check.png "[Apply]") (Apply)  
+    Click this button to apply the entered connection name. This button is only displayed if the connection name has not yet been confirmed. 
+
+- *Driver*  
+    Click the drop-down list and select the *BigCommerce* driver. All installed drivers are displayed. The *Credentials* section is displayed below the drop-down list.
+
```

Comment:
In den anderen Verbindungen gibt es eine Info dazu. Ist sie auch hier zutreffend?

[Info] Drivers are licensed and must be acquired via the app store or the corresponding e-commerce partner platform. The applicable driver credentials to establish the connection are obtained when acquiring the corresponding license.

Reply:
done